compiler/rustc_middle/src/ty/context.rs

```diff
@@ -2230,6 +2230,10 @@ impl<'tcx> TyCtxt<'tcx> {
             })
         )
     }
+
+    pub fn is_solver_next(self) -> bool {
```

Comment:
i somewhat dislike that method name. Would prefer `fn trait_solver_next` or `fn use_trait_solver_next`.